client/src/main.js

```diff
@@ -6,8 +6,12 @@ import './registerServiceWorker'
 import './quasar'
 import GAuth from 'vue-google-oauth2'
 import { clientId } from './config/config.js'
+import moment from 'moment'
 import VueObserveVisibility from 'vue-observe-visibility'
 Vue.use(VueObserveVisibility)
+
+Vue.prototype.$moment = moment
```

Comment:
Added to the global scope but as far as I see it - you still never used it (aka $moment references)